### PR TITLE
Improved toHaveState's support for nested objects to display nested o…

### PR DESCRIPTION
**Pull request by andreysaleba**
…bjects contents instead of [Object object] useless string

### CODE REVIEW

**Comment by andreysaleba** [comment]
#145

File: package.json [hunk 1]
```diff
@@ -78,5 +78,8 @@
       "<rootDir>/packages/enzyme-matchers/src"
     ],
     "verbose": true
+  },
+  "dependencies": {
+    "circular-json-es6": "^2.0.1"
```
Comment (blainekasten) [inline package.json]: This needs to be a dependency of packages/enzyme-matchers, not this root package.json. Otherwise it won't get bundled up correctly.

**Comment by andreysaleba** [comment]
Moved the dependency out of the root package.json

**Comment by blainekasten** [comment]
This looks great! But I need you to rebase down to 1 commit please! 🙏🙏🙏

**Comment by andreysaleba** [comment]
@blainekasten yeah, is it better now?

**Comment by blainekasten** [comment]
Hmm. no. It looks like the rebase didn't go quite correct. I can tell because your diff now includes things already in master. You should be doing a rebase with the squash feature. Or since your changes are relatively small, you could just redo your work off of latest master real quick and put up a new PR.

**Comment by blainekasten** [comment]
@andreysaleba still waiting for a good rebase! I'm looking forward to merging this PR and cutting a new release!

**Pull request closed without merging**